heroic-core/src/main/java/com/spotify/heroic/http/metadata/MetadataResource.java

```diff
@@ -113,12 +114,16 @@ public void addSeries(@Suspended final AsyncResponse response, final Series seri
     public void getTimeSeries(
         @Suspended final AsyncResponse response, final MetadataQueryBody request
     ) {
-        final RequestCriteria c = toCriteria(request::getFilter, request::getRange,
-            () -> OptionalLimit.of(request.getLimit().orElse(MetadataQueryBody.DEFAULT_LIMIT)));
+        RequestCriteria criteria = toCriteria(
+            request::getFilter,
+            request::getRange,
+            () -> OptionalLimit.of(request.getLimit().orElse(MetadataQueryBody.DEFAULT_LIMIT))
+        );
 
-        httpAsync.bind(response, query
-            .useDefaultGroup()
-            .findSeries(new FindSeries.Request(c.getFilter(), c.getRange(), c.getLimit())));
+        FindSeries.Request seriesRequest = new FindSeries.Request(
+            criteria.getFilter(), criteria.getRange(), criteria.getLimit(), Features.DEFAULT);
```

Comment:
this is effectively hardcoding this endpoint to the default features right? i think we only care about testing the metric path anyway

Reply:
Yeah, I can add it to that path too. Just needed to have something in there for the updated constructor.

Reply:
I don't think it matters too much - once we test this on the query metric path, it will likely become the default or only choice.

Reply:
Cool - there's no feature support in any `/metadata` queries right now, so its actually kind of annoying to add it in. I'll skip it.